pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -410,63 +397,63 @@ func listEqualUnordered(a []string, b []string) bool {
 	return true
 }
 
-// update the node associated with the connection, after receiving a a packet from envoy.
-func (s *DiscoveryServer) initConnectionNode(node *core.Node, con *XdsConnection) error {
+// update the node associated with the connection, after receiving a a packet from envoy, also adds the connection
+// to the tracking map.
+func (s *DiscoveryServer) initConnection(node *core.Node, con *XdsConnection) (func(), error) {
+	initialized := false
```

Comment:
redundant

Reply:
did not understand? What is redundant?

Reply:
This does nothing since we always set it on line 406

Reply:
Oh. Sorry. did not realize that. Will correct that in follow-up